@here/harp-materials/lib/ShaderChunks/ExtrusionChunks.ts

```diff
@@ -12,7 +12,10 @@ uniform float extrusionRatio;
 varying vec4 vExtrusionAxis;
 `,
     extrusion_vertex: `
-transformed = transformed - extrusionAxis.xyz + extrusionAxis.xyz * extrusionRatio;
+// animate only if the point is above the ground
+if(dot(position,vec3(extrusionAxis))>0.0){
```

Comment:
I still don't get why we have to do this. Either this is a data issue then we should report it to map compilation but most likely it is a bug in our geometry generation code or we should do some clamping there instead of doing it in the shader

Reply:
You're right, I will rework it to fix the geometry instead, as this `extrusionAxis` is telling points on the ground to go below it.